src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,36 @@
+import errorIcon from "@assets/images/error.svg";
+import warningIcon from "@assets/images/warning.svg";
+import { Tooltip } from "@saleor/macaw-ui";
+import React from "react";
+import SVG from "react-inlinesvg";
+
+export type BadgeVariant = "warning" | "error";
```

Comment:
I think since we have exactly same color logic in our `StatusChip` component (and purposefully) it could be more unified using `status` instead of `variant` from status chip
```
export enum StatusType {
  NEUTRAL = "neutral",
  ERROR = "error",
  ALERT = "alert",
  SUCCESS = "success"
}

Reply:
It cannot be unified at this moment, because StatusBadge uses Tooltip component from MacawUI, which takes variant as an argument. Tooltip requires variants to be "error", "info",  "success", "warning".